app/views/services/utilities_index.html.erb

```diff
@@ -1,41 +1,41 @@
 <h1 class="center pad-25-bottom">Utilities</h1>
 
 <hr>
-
-<div class="container">
+<center>
+<div class="container pad-25-top">
   <div class="row">
     <div class="col-sm pad-25-bottom">
       <div class="btn-group2">
         <a href="<%= "/#{params[:location]}/utilities/electricity" %>" class="btn button-service">
-          <%= image_tag "electricity", {size: "110" } %>
-          <h2>Electricity</h2>
+          <%= image_tag "electricity" ,{size: "110" } %>
+            <h2>Electricity</h2>
```

Comment:
h2 tags shouldn't be indented here (done through the rest of the file as well)

Reply:
Done